### ISSUE TITLE
Character frequency display

### ISSUE THREAD

**Issue by batterseapower** [batterseapower/pinyin-toolkit#139]
When entering an expression into the fact editor it would be nice if there was some visual indication of how frequently it occurs (as on e.g. Nciku). This might prevent you filling up your deck with mostly-useless vocabulary.

Not sure how this would be implemented yet - obviously some sort of internet lookup is a possibility.

**Comment by Nick3C** [comment]
I think we should add a little display below the toolbar of the add window that lists various pieces of information about the word/character.

In terms of implmenting this obvious possibilities are using a google search to obtain the hitcount (or doing so on nciku, although it will be less reliable for rare words). We could also look at using the data from the chinese examples plugin (as it is local). We could query that for frequency too.

The best way would probably be to add a figure to the new toolbar showing google frequency count and nciku freq count if we have online access; and if neither is available show just the local file data. For single characters we could use local data for character frequency instead of querying word frequency as this is more important [we still need to try and get for the first 10,000 characters, not just our current 3,500 as well]

Other features on such a bar could include something I have mentioned before:little green/red flags that show whether it is in certain on-line dictionaries [probably will meaning over-write fill if the user clicks on them]. Additionally we could have "hsk level" indicator using the word not character (MDBG can be queried for this).

integration into the main window and other stats areas might be a long term aim.

**Comment by batterseapower** [comment]
Yes, HSK indication would also be good. A general side bar that could also do what the link bar intended would be best.

I've just found that Unihan has character frequency data, which might come in handy (but no cjklib support yet).
